router.go

```diff
@@ -88,6 +88,11 @@ import (
 // wildcards (path variables).
 type Handle func(http.ResponseWriter, *http.Request, Params)
 
+// ServeHTTP calls function type of Handle.
```

Comment:
Please align this with the comment for [`http.HandlerFunc.ServeHTTP`](https://golang.org/pkg/net/http/#HandlerFunc.ServeHTTP): `ServeHTTP calls h(w, r, ps)`.

Reply:
Thank you for reviewing my code!
I've just fixed the comment that you have pointed out.